src/components/Payments/index.tsx

```diff
@@ -0,0 +1,52 @@
+import { usePayments } from "../../hooks/usePayments";
+import CardForm from "./CardForm";
+import CardList from "./CardList";
+import Completed from "./Completed";
+
+const Payments = () => {
+  const {
+    step,
+    setStep,
+    cardList,
+    newCardInfo,
+    clearCardInfo,
+    handleCardInputChange,
+    handleCardTypeClick,
+    handleCardAddClick,
+    handleCardNicknameAddClick,
+  } = usePayments();
+
+  switch (step) {
+    case 1:
+    case 2:
+      return (
+        <CardForm
+          newCardInfo={newCardInfo}
+          step={step}
+          setStep={setStep}
+          handleCardInputChange={handleCardInputChange}
+          handleCardTypeClick={handleCardTypeClick}
+          handleCardAddClick={handleCardAddClick}
+        />
+      );
+    case 3:
+      return (
+        <Completed
+          newCardInfo={newCardInfo}
+          setStep={setStep}
+          handleCardInputChange={handleCardInputChange}
+          handleCardNicknameAddClick={handleCardNicknameAddClick}
+        />
+      );
+    default:
+      return (
+        <CardList
+          clearCardInfo={clearCardInfo}
+          cardList={cardList}
+          setStep={setStep}
+        />
+      );
```

Comment:
switch-case문을 작성할 때 default문을 작성하는 건 매우 좋은 습관인데요. switch-case 문 안에 존재하는 다른 case에 비해서 어떤 의미인지가 잘 드러나지 않는거 같아요.

해당 컴포넌트를 return 하게 되는 조건도 같이 작성해보시면 어떨까요?

```suggestion
    case "something":
    default:
      return (
        <CardList
          clearCardInfo={clearCardInfo}
          cardList={cardList}
          setStep={setStep}
        />
      );
```

Reply:
default가 필요하다는 생각만하고 어떤케이스에 default가 필요한지까지 명시해줄 생각을 못했었는데 이번 기회를 통해서 default가 어떨때 이루어져야 하는지도 적어주면 좋다라는걸 배웠습니다!